inverse_covariance/inverse_covariance.py

```diff
@@ -12,6 +13,9 @@ def _init_coefs(X, method='corrcoef'):
     elif method == 'cov':
```

Comment:
I guess we'll need to document it in an issue when we find it. I remember there being some cases where we could not get reasonable results with one of corrcoef or cov, a scaling issue or something